udata/templates/territories/territory.html

```diff
@@ -46,5 +46,5 @@
 {% block extra_js %}
     <script src="{{ static('vue-common.js') }}"></script>
     <script src="{{ static('territory.js') }}"></script>
-    <script src="{{ static('widgets.js', _burst=False) }}" id="udata" async defer onload="udataScript.load()"></script>
+    <script src="{{ static('widgets.js', _burst=False) }}" id="udata" async defer onload="udataScript.loadDatasets()"></script>
```

Comment:
I was refering to:
- https://github.com/opendatateam/udata/blob/master/js/templates/widgets/integrate-popover.hbs
- https://github.com/etalab/udata-gouvfr/blob/master/udata_gouvfr/templates/faq/developer.html#L57-L61